src/main/java/com/northconcepts/datapipeline/examples/cookbook/GenerateMySqlDdlProgrammatically.java

```diff
@@ -0,0 +1,41 @@
+package com.northconcepts.datapipeline.examples.cookbook;
+
+import com.northconcepts.datapipeline.jdbc.sql.SqlPart;
+import com.northconcepts.datapipeline.sql.mysql.*;
+
+public class GenerateMySqlDdlProgrammatically {
+    public static void main(String[] args) {
+        MySqlPart createUsersTable = new CreateTable("employee")
+                .addColumn(new CreateTableColumn("id", TableColumnType.INT).setAutoIncrement(true).setPrimaryKey(true))
+                .addColumn(new CreateTableColumn("name", TableColumnType.TEXT).setNullable(false))
+                .addColumn(new CreateTableColumn("email", TableColumnType.TEXT).setUnique(true))
```

Comment:
Let's use `vachar` with a set length instead of `text` for most string fields unless they should really be variable length like `description` below.  `text` fields should be use sparingly since they come with many downsides.

Reply:
done